applications/nrf5340_audio/src/nrf5340_audio_common.c

```diff
@@ -26,5 +27,12 @@ int nrf5340_audio_common_init(void)
 		return ret;
 	}
 
+	/* Use this to turn on 128 MHz clock for cpu_app */
```

Comment:
Nitpick: Maybe we could move this part to the beginning of the code for speed up a tiny little bit.

Reply:
That's a good idea! I put it in this module as it is common for all the applications, but maybe for this reason it's better to initialize the clock at the beginning of each `main.c` file? I'm not sure if the `nrf5340_audio_common_init()` call can come before the other init's 🤔

Reply:
I think it can, as mentioned in a comment earlier.

Reply:
Moved to top of function, but not to top of `main()` because of blockers explained in other comment